addons/mail/static/src/discuss/core/common/discuss_channel_model.js

```diff
@@ -70,6 +70,13 @@ export class DiscussChannel extends Record {
         return def;
     }
 
+    get allowCalls() {
+        return (
+            !this.isTransient &&
+            this.typesAllowingCalls.includes(this.channel_type) &&
+            !this.correspondent?.persona.eq(this.store.odoobot)
```

Comment:
Dangerous negation on optional chaining. Would replace to

`this.correspondent?.persona.notEq(this.store.odoobot)`